mitosheet/src/mito/Mito.tsx

```diff
@@ -306,6 +306,7 @@ export const Mito = (props: MitoProps): JSX.Element => {
                             }
                         }
                     })
+                    void mitoAPI.log('user_edited_code_cell')
```

Comment:
Would be nice if this log would capture what was edited here - can we log the old code and the new code?